src/shared/Elements/Button.tsx

```diff
@@ -10,17 +10,18 @@ export interface IButtonProps {
 export const Button = styled.button<IButtonProps>`
   background-color: ${(props) =>
     props.secondary ? props.theme.colors.grey : props.theme.colors.primary};
-  font-size: 14px;
+  font-size: 16px;
```

Comment:
This is a global change, for all buttons on the dashboard. Have we confirmed that this doesn’t make other usages of button look weird / bad?

Reply:
we're updating the UI completely so this will be addressed once all the other implementations are started! based on the design, all font sizes are 16 px !